SPECS/python-lxml/python-lxml.spec

```diff
@@ -40,46 +35,34 @@ Python 3 version.
 %prep
 %setup -q -n lxml-%{version}
 %patch0 -p1
-rm -rf ../p3dir
-cp -a . ../p3dir
 
 %build
-python2 setup.py build
-pushd ../p3dir
 python3 setup.py build
-popd
 
 %install
-python2 setup.py install --skip-build --root %{buildroot}
-pushd ../p3dir
 python3 setup.py install --prefix=%{_prefix} --root=%{buildroot}
-popd
 
 %check
 export LC_ALL=en_US.UTF-8
 export LANGUAGE=en_US.UTF-8
 make test
-pushd ../p3dir
-make test
-popd
 
 %clean
 rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%license LICENSES.txt
-%{python2_sitelib}/lxml/*
-%{python2_sitelib}/lxml-%{version}-py2.7.egg-info
 
 %files -n python3-lxml
 %defattr(-,root,root,-)
+%license LICENSES.txt
 %{python3_sitelib}/*
 
 %changelog
-* Sat May 09 00:20:50 PST 2020 Nick Samson <nisamson@microsoft.com> - 4.2.4-6
-- Added %%license line automatically
-
+*   Wed Aug 26 2020 Thomas Crain <thcrain@microsoft.com> 4.2.4-7
+-   Remove python2 support.
```

Comment:
Please also check the license and add the `License verified.` entry to the changelog, since we're adding the `%license` macro.

Btw. thank you for doing that (either to you or Thomas). :)

Reply:
Done